nuxeo-lambda-image-conversion/src/main/java/org/nuxeo/lambda/image/conversion/listener/LambdaResponseListener.java

```diff
@@ -38,8 +30,19 @@
 import org.nuxeo.ecm.core.work.api.WorkManager;
 import org.nuxeo.ecm.platform.picture.PictureViewsGenerationWork;
 import org.nuxeo.lambda.core.LambdaService;
+import static org.nuxeo.lambda.core.LambdaService.LAMBDA_RESPONSE_KEY;
+import static org.nuxeo.lambda.core.LambdaService.PARAMETERS_KEY;
+import static org.nuxeo.lambda.image.conversion.common.Constants.DOC_ID_PROP;
+import static org.nuxeo.lambda.image.conversion.common.Constants.REPOSITORY_PROP;
```

Comment:
Imports are not well organised. We should have:
- static
- java
- javax
- org